pages/linux/goldeneye.md

```diff
@@ -0,0 +1,24 @@
+# goldeneye
+
+> A HTTP DoS test tool.
+> More information: <https://github.com/jseidl/GoldenEye>.
+
+- Test a specific website:
+
+`./goldeneye.py {{url}}"`
+
+- Test a specific website with 100 user agents and 200 concurrent sockets:
+
+`./goldeneye.py {{url}} -u {{100}} -s {{200}}"`
```

Comment:
```suggestion
`./goldeneye.py {{url}} --useragents {{100}} --sockets {{200}}"`
```
We prefer longer arguments in tldr, whenever possible we add mnemonics in command descriptions to highlight the short arguments.